cdist/conf/type/__install_coreos/gencode-remote

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -e
+
+device=$(cat "${__object:?}/parameter/device")
+ignition=$(cat "${__object}/parameter/ignition")
+
+cat <<EOF
+ignition_file="\$(mktemp /tmp/cdist.XXXXXXXXXX)"
+
+base64 -d > "\${ignition_file}" << eof
+$(base64 "${ignition}")
+eof
+
+coreos-install -d "${device}" \
```

Comment:
Maybe mention the requirement of having coreos-install in the running preos in the man page

Reply:
I've updated the man page.